### ISSUE TITLE
TypeError:__init__() got an unexpected keyword argument 'presistent_workers'

### ISSUE THREAD

**Issue by 55998** [Project-MONAI/research-contributions#15]
**Describe the bug**
I use the UNETR/BTCV code to complete multi organ segmentation, but in Dataloader, I get this Error:
TypeError:__init__() got an unexpected keyword argument 'presistent_workers'. It exist in .../monai/data/dataloader.py, line 87, in __init__ **kwargs

**To Reproduce**
1. Go to 'UNETR/BTCV'
2. Install 'monai==0.7.0 nibabel==3.1.1 tqdm==4.59.0 einops==0.3.2 tensorboardx==2.1'
3. Run commands 'python main.py
--batch_size=1
--logdir=unetr_pretrained
--optim_lr=1e-4
--lrschedule=warmup_cosine
--infer_overlap=0.5 
--save_checkpoint
--data_dir=/dataset/dataset0/
--pretrained_dir='./pretrained_models/'
--pretrained_model_name='UNETR_model_best_acc.pth'
--resume_ckpt'

**Expected behavior**
Start the train correctly.

**Screenshots**
![image](https://user-images.githubusercontent.com/52224876/139210253-b5aec7d5-9ac4-4fa7-b8bb-24caf8fe8777.png)

**Environment (please complete the following information):**
 - OS   ubuntu16.04
 - Python version   Python 3.6
 - MONAI version [e.g. git commit hash]   0.7.0
 - CUDA/cuDNN version   10.2
 - GPU models and configuration  Geforce Nvidia RTX 2080ti 11G

**Additional context**

**Comment by 55998** [comment]
Hi @ahatamiz 

Thank you for your reply. Now I can correctly use this code.

But I got a new issue. I use two Nvidia Geforce RTX 2080Ti, and torch==1.9.1, the data is same with readme, and training Parameters is 
--feature_size=32 
--batch_size=1
--logdir=unetr_test
--optim_lr=2e-4
--lrschedule=warmup_cosine
--infer_overlap=0.5 
--save_checkpoint
--data_dir=/dataset/dataset0/
--distributed
I set the max_epochs is 3000. But when finish the training, I got the Final traing 2999/2999 loss:0.5940, Final validation 2999/2999 acc 0.8024047, Training Finished! **Best Accuracy: 0.0**
So, witch problem with it?

![1635560067(1)](https://user-images.githubusercontent.com/52224876/139517008-75517fe7-4346-4462-881f-07805e14d340.png)

**Comment by 55998** [comment]
Hi @ahatamiz 

I Used the best_model to test, I get the AVG_DICE  was 0.79; Without the pre_training model, I set 3000epochs, and the final validation ACC was 0.80
The data display seems a little low. What may be the reason? The data set I use is abdomen CT raw_data in btcv challenge.

**Comment by hnjzbss** [comment]
> Hi @ahatamiz
> 
> Thank you for your reply. Now I can correctly use this code.
> 
> But I got a new issue. I use two Nvidia Geforce RTX 2080Ti, and torch==1.9.1, the data is same with readme, and training Parameters is --feature_size=32 --batch_size=1 --logdir=unetr_test --optim_lr=2e-4 --lrschedule=warmup_cosine --infer_overlap=0.5 --save_checkpoint --data_dir=/dataset/dataset0/ --distributed I set the max_epochs is 3000. But when finish the training, I got the Final traing 2999/2999 loss:0.5940, Final validation 2999/2999 acc 0.8024047, Training Finished! **Best Accuracy: 0.0** So, witch problem with it?
> 
> ![1635560067(1)](https://user-images.githubusercontent.com/52224876/139517008-75517fe7-4346-4462-881f-07805e14d340.png)



> 嗨@ahatamiz
> 
> 感谢你的回复。现在我可以正确使用这段代码了。
> 
> 但是我遇到了一个新问题。我用了两个Nvidia Geforce RTX 2080Ti，torch==1.9.1，数据和readme一样，训练参数是 --feature_size=32 --batch_size=1 --logdir=unetr_test --optim_lr=2e-4 - -lrschedule=warmup_cosine --infer_overlap=0.5 --save_checkpoint --data_dir=/dataset/dataset0/ --distributed 我设置的max_epochs是3000。但是当完成训练时，我得到了Final traing 2999/2999 loss:0.5940, Final验证 2999/2999 acc 0.8024047，培训完成！**最佳准确度：0.0** 那么，有问题吗？
> 
> ![1635560067(1)](https://user-images.githubusercontent.com/52224876/139517008-75517fe7-4346-4462-881f-07805e14d340.png)

How did you solve this problem?

**Comment by ahatamiz** [comment]
Hi @55998  

I was able to reproduce this issue. 

Based on this, I have submitted a new pull request (#18) that addresses this issue. Please re-try once it's merged. 

Thanks
